testing/kuttl/kuttl-test.yaml

```diff
@@ -2,6 +2,7 @@ apiVersion: kuttl.dev/v1beta1
 kind: TestSuite
 testDirs:
 - testing/kuttl/e2e-generated/
+- testing/kuttl/e2e-generated-other/
```

Comment:
I don't think we would want to run this as part of the standard `make check-kuttl` command. I think having it defined here would try to run it

Reply:
:thinking: I can see that being an issue. I'll add a note in the README with an alternate way to run the test.